Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -500,25 +505,33 @@ class ArbitraryMipmapDetector
 
     // This is the average per-pixel, per-channel difference in percent between what we
     // expect a normal blurred mipmap to look like and what we actually received
-    constexpr auto THRESHOLD_PERCENT = 35.f;
+    constexpr auto THRESHOLD_PERCENT = 4.5f;
+
+    auto* src = downsample_buffer;
```

Comment:
`const auto*`

Reply:
This actually can't be `const auto*` because `src` and `dst` are swapped to double-buffer, so they both are written to at different times.